internal/service/eks/addon_test.go

```diff
@@ -246,6 +246,38 @@ func TestAccEKSAddon_serviceAccountRoleARN(t *testing.T) {
 	})
 }
 
+func TestAccEKSAddon_configurationValues(t *testing.T) {
```

Comment:
We should also include a negative test to ensure that if the user supplies an invalid ConfigurationValue string that the errors are handled correctly in the provider.